submissions/GryphHacksVidDownloader/README.md

```diff
@@ -0,0 +1,27 @@
+# ytgui
+
+A youtube downloader for any Gryph Hackers
+
+## Usage
+
+Must instal pytube by typling "pip3 install pytube" into terminal to run
```

Comment:
spelling
```suggestion
Must instal pytube by typling `pip3 install pytube` into terminal to run
```

using code blocks is a nice way of making the commands stand out in a line of text